rally_ovs/plugins/ovs/scenarios/sandbox.py

```diff
@@ -159,10 +160,16 @@ def _create_sandbox(self, sandbox_create_args):
 
             cmds = []
             for host_ip in host_ip_list:
-                cmd = "./ovs-sandbox.sh --ovn --controller-ip %s \
-                             --host-ip %s/%d --device %s" % \
-                         (controller_ip, host_ip, sandbox_cidr.prefixlen,
-                                net_dev)
+                if ssl:
+                    cmd = "./ovs-sandbox.sh --ovn --controller-ip %s \
+                                 --host-ip %s/%d --device %s --ssl" % \
+                          (controller_ip, host_ip, sandbox_cidr.prefixlen,
+                           net_dev)
+                else:
+                    cmd = "./ovs-sandbox.sh --ovn --controller-ip %s \
+                                 --host-ip %s/%d --device %s" % \
+                             (controller_ip, host_ip, sandbox_cidr.prefixlen,
+                                    net_dev)
```

Comment:
please abstract out the different part only (--ssl)

Reply:
sure